src/Shared/AppLoadingView.xaml.cs

```diff
@@ -259,7 +259,18 @@ async void OutOfBox_CoreInstanceRegistered(object? sender, CoreInstanceEventArgs
 
                 core.CoreStateChanged += Core_CoreStateChanged;
 
-                await core.InitAsync();
+                try
+                {
+                    await core.InitAsync();
+                }
+                catch (OperationCanceledException)
+                {
+                    // TODO: Does the core need to be disposed?
+                    // TODO: Navigate back to OOBE proper. As is, it just stays
```

Comment:
What is the proper way to navigate to the first page in the OOBE navigation stack that isn't from a core? Or should we always navigate to the core selection page?

Reply:
The core should be unregistered with the `coreManagementService`, and the UI will pick up on that. You should have access to an instance of this.